packages/flutter/lib/src/material/outlined_button.dart

```diff
@@ -299,8 +299,8 @@ class OutlinedButton extends ButtonStyleButton {
   /// * `side` - BorderSide(width: 1, color: Theme.colorScheme.onSurface(0.12))
   /// * `shape` - RoundedRectangleBorder(borderRadius: BorderRadius.circular(4))
   /// * `mouseCursor`
+  ///   * enabled - WidgetStateMouseCursor.adaptiveClickable
   ///   * disabled - SystemMouseCursors.basic
-  ///   * others - SystemMouseCursors.click
```

Comment:
Isn't this simply
```
  /// * `mouseCursor`: [WidgetStateMouseCursor.adaptiveClickable]
```
Since the WidgetState part is about the enabling state?


There are a few other similar cases throughout the PR.